doc/source/dev/contributor/meson_advanced.rst

```diff
@@ -86,3 +86,228 @@ to run the ``scipy.linalg`` tests for builds with both compilers, do::
     python dev.py -s linalg  # will run the tests for the GCC build
     python dev.py --build-dir build-clang -s linalg  # will run the tests for the Clang build
 
+
+Introspecting build steps
+=========================
+
+When you have an issue with a particular Python extension module or other build
+target, there are a number of ways to figure out what the build system is doing
+exactly. Beyond looking at the ``meson.build`` content for the target of
+interest, these include:
+
+1. Reading the generated ``build.ninja`` file in the build directory (available
```

Comment:
Did you mean to complete this sentence after "available"?

Reply:
good catch, fixed